x-pack/plugins/endpoint/public/applications/endpoint/models/policy_details_config.ts

```diff
@@ -43,3 +43,33 @@ export function clone(policyDetailsConfig: UIPolicyConfig): UIPolicyConfig {
    */
   return clonedConfig as UIPolicyConfig;
 }
+
+/**
+ * Returns value from `configuration`
+ */
+export const getIn = (a: UIPolicyConfig) => <Key extends keyof UIPolicyConfig>(key: Key) => <
```

Comment:
this works to keep us type compliant when we have different combinations of keys?

Reply:
yeah this was to make sure the types were still getting passed through correctly